Button.cpp

```diff
@@ -0,0 +1,91 @@
+// Button.h
+// Sheng Fang, Rabiya Majeed, & Zulaikha Zakiullah
+// This instantiates the button class
+
+#include "Button.h"
+
+Button::Button(int w, int h) {
+
+	width = w;
+	height = h;
+
+	position.x = 0;
+	position.y = 0;
+
+	currentSprite = ButtonSprite::BUTTON_SPRITE_MOUSE_OUT;
+}
+
+Button::~Button() {}
+
+// Setters
+
+void Button::setPosition(int x, int y) {
+	position.x = x;
+	position.y = y;
+}
+
+void Button::setWidth(int w) {
+	width = w;
+}
+
+void Button::setHeight(int h) {
+	height = h;
+}
+
+// Getters 
+
+int Button::getPosition() {
+	return position.x, position.y; 
+}
+
+int Button::getWidth() {
+	return width;
+}
+
+int Button::getHeight() {
+	return height;
+}
+
+void Button::handleEvent(SDL_Event* e) {

```

Comment:
To follow naming convention, can this be handle_event? Unless this is a pre-defined function/interface(?) in SDL.